gateway/src/project.rs

```diff
@@ -1497,7 +1497,7 @@ impl Service {
                 .method(Method::PUT)
                 .uri(uri)
                 .header(AUTHORIZATION, format!("Bearer {}", jwt))
-                .header(X_SHUTTLE_ACCOUNT_NAME.clone(), "gateway")
+                .header(X_SHUTTLE_USER_ID.clone(), "gateway")
```

Comment:
The user id for gateway won't be gateway right?

Reply:
For those special accounts, we should also update their user id to be non-standard.